scripts/pulse-results/README.md

```diff
@@ -0,0 +1 @@
+this directory contains the json output from `process_a11y` script
```

Comment:
I don't believe this directory should be in version control. Please use `utils.mkdir_p()` to create it on demand from the script.